### DISCUSSION TITLE
Importing svg files by using the swc compiler.

### DISCUSSION THREAD

**Discussion by navanshu** [TypeStrong/ts-node#1590] [Q&A]
I tried importing the SVG file in typescript but it threw an error as SWC cannot parse SVG.
I am thinking of using the SWC core experimental transpiler library as it presents a create function to ts-node and as it is passed an SVG file I just read it as a string and provide the same. Is there an alternative way as I couldn't find any information regarding this?
Hope someone can help me.

**Comment by cspotcode** [comment]
Reading files on node is typically accomplished using the filesystem APIs available via `import * as fs from 'fs';`
